Rakefile

```diff
@@ -415,8 +415,16 @@ namespace :native_dev do
   CLEAN.concat(NATIVE_CLEAN)
 end
 
+task :standard_profiling do
```

Comment:
What about this?

```
namespace :standard do
  task :profiling do
  
task :standard => [:'standard:profiling']
```

Reply:
My thinking was that `standard` already includes their own rake tasks, so I wanted to avoid clashing with the built-in ones.

By using `standard_profiling`, I was hoping to make it clear that this is our custom config for profiling only, and the idea would be that we would remove it once we're happy with using standard everywhere and go back to the built-in standard rake tasks.

Reply:
I ventured as much, but I don't find it very clear that it's our task. It'd be clearer if it were `namespace`d `:datadog`. Also having a `desc` can make that clear, plus make it discoverable through `rake -T` ;)

My rationale is that since the standard rake task is not there yet and it's due for removal once it is then it's totally OK.

Reply:
Also then you can have:

```
task default: ['rubocop', 'standard', 'typecheck', 'spec:main']
```

And when we pivot to the standard task it's a drop-in replacement.

Which makes me think that unless I missed something, as is profiling is not lint-checked in CI anymore? ([see here](https://github.com/DataDog/dd-trace-rb/blob/7de53929e489ff6025855eb263bf79d82be48f43/.circleci/config.yml#L161))

Reply:
Fixed in a3588e19905b3a7d272a2e0cc69997e0a55247ee and abc3ef4fa85a7e80f7538337800f79dcb0af40e7 .

I've moved standardrb to a separate namespace + added descriptions and also updated the CI configuration. The CI part in particular I was under the (mistaken) expectation that CI also used the default task, thanks for pointing out it doesn't.